recipes/mender-client/debian-master/control

```diff
@@ -8,13 +8,15 @@ Build-Depends: debhelper (>= 9)
 Package: mender-client
 Conflicts: mender
 Architecture: any
-Depends: libc6 (>= 2.12), libglib2.0-0 (>=2.50), liblzma5 (>= 5.1), libssl1.1 (>= 1.1), libffi6 (>= 3.2)
+Depends: ${shlibs:Depends}, ${misc:Depends}
+# debian bullseye Depends: libc6 (>= 2.14), libglib2.0-0 (>= 2.26.0), liblzma5 (>= 5.1.1alpha+20120614), libssl1.1 (>= 1.1.1)
+# debian buster Depends: libc6 (>= 2.14), libglib2.0-0 (>= 2.26.0), liblzma5 (>= 5.1.1alpha+20120614), libssl1.1 (>= 1.1.1)
```

Comment:
Open question, what should we do with old recipes?

As it is `mender-deb-package` now it will always expect these commended lines for armhf builds, but that is not true on `recipes != master` so bugfix releases might get broken Dependencies.

Should we port this back to all recipes or just check in for existence of these lines in `mender-deb-package` before post-processing them (calling `sed`)

Reply:
This we don't need to solve "today", we only need to solve it before we build the bugfix releases (some time next week).